creator-node/src/middlewares.js

```diff
@@ -120,7 +120,7 @@ async function triggerSecondarySyncs (req) {
 }
 
 /** Retrieves current FQDN registered on-chain with node's owner wallet. */
-async function _getOwnEndpoint (req) {
+async function getOwnEndpoint (req) {
```

Comment:
why rename this? it's a NIT but _internal functions seemed kind of nice

Reply:
i export this function so i removed the underscore. also this isn't the right place for this file, feels wrong importing a function from middleware to call, but wanted to keep this pr light